doc/modelling/unit_operations/population_balance_model.rst

```diff
@@ -0,0 +1,154 @@
+.. _pbm_model:
+
+Population balance model (PBM)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The population balance equation is a particle-number continuity equation which describes the evolution of the number density :math:`n` of the particles in the time and space domains. 
+The particles of interest have both internal and external coordinates: the internal coordinate can be chosen as any property of the particles such as the particle size or volume 
+while the external coordinate can be a characteristic dimension of the reactor itself, including the axial position. The current release considers the nucleation, growth and growth rate dispersion. 
+The equations can be solved in CSTR or DPFR (LRM) to model batch or continuous processes. Aggregation and fragmentation processes will be included later. 
```

Comment:
Can't they be used for any unit operation? I could still use them in the GRM if I wanted to, right?

Reply:
Implementation-wise yes, but using it in GRM does not make any sense in practice as nobody grows their crystals in columns with beads.

Reply:
It's not so much about growing crystals. However, precipitation of proteins can play a role in chromatography, albeit a usually a negative side effect. Might still be interesting to model for some people.

Reply:
We should at least mention it as an option. Sometimes people come up with applications we never thought of when implementing the code.